.github/workflows/upload-docs.yml

```diff
@@ -0,0 +1,48 @@
+name: Upload Docs Android
+
+on:
+  push:
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+'
+      - 'test-doc-*'
+
+jobs:
+  upload-docs:
+
+    runs-on: macos-latest
+
+    steps:
+    - uses: actions/checkout@v4
+    - uses: actions/setup-java@v4
+      with:
+        java-version: '17'
+        distribution: 'temurin'
+    - name: Generate docs
+      run: |
+        ./gradlew :PrebidMobile:combinedJavadoc
+        cp -r PrebidMobile/build/combinedJavadoc docs_new
+    - name: Checkout repository
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+        ref: docs
+        clean: false
+    - name: Commit documentation changes
+      run: |
+        git config --local user.email "41898282+github-actions[bot]@users.noreply.github.com"
+        git config --local user.name "github-actions[bot]"
+        git status
+        TAG_NAME=${{ github.ref }}
+        TAG_NAME=${TAG_NAME#refs/tags/}
+        echo "Current tag: $TAG_NAME"
+        ls -l
+        rm -rf docs
```

Comment:
Seems like there ought to be a check here to make sure that docs_new actually exists and is of a certain size before removing the current `docs` dir. e.g. use `du -s docs` and abort the operation if less than some reasonable size.